packages/core/src/odata/v4/request-builder/create-request-builder.ts

```diff
@@ -1,6 +1,6 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
-import { errorWithCause } from '@sap-cloud-sdk/util';
+import { errorWithCause, MapType } from '@sap-cloud-sdk/util';
```

Comment:
I compared this once with the v2 create request builder. I did the same also for the get all and get by key and they are also very similar. Please create a follow up ticket that we do not forget to reduce this code duplication. Obvious way would be base classes but mixin could be a more TS way to do that.

Reply:
see comment above.